app/shared/src/services/__tests__/Translate.test.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import { NativeModules } from 'react-native';
```

Comment:
My OCD is going crazy every time I see import under `// @flow` without an empty line. I know - it should not be in the code review but in eslint somehow... 😄